site3/website/docs/admin/http.md

```diff
@@ -462,6 +468,38 @@ Currently all the HTTP endpoints could be divided into these 5 components:
        }
       ```
 
+### Endpoint: /api/v1/bookie/state/readonly
+1. Method: GET
+    * Description: Get bookie readOnly state.
+    * Response:  
+    
+        | Code   | Description |
```

Comment:
The response should contains the `readOnly` property

Reply:
OK